server/analytics/common.py

```diff
@@ -22,6 +22,8 @@
 
 logger = logging.getLogger(__name__)
 
+MAX_TERMS_SIZE = 2_147_483_647
```

Comment:
I think we could use some number in 1k range, if you have aggregation with more results it's pretty much useless anyhow, and not sure how the UI would handle it

Reply:
Agree'd. I've set it to 1,000 (and even then displaying 1,000 items in a chart is not readable).